### DISCUSSION TITLE
Use signedRoute,unable to access after changing the domain name

### DISCUSSION THREAD

**Discussion by krcharen** [laravel/framework#47583] [Q&A]
My current website domain name is A. 
After using the signedRoute function to generate a signed permanent link, when I change the website domain name to B, I cannot access the previously generated permanent link at this time.

**A domain** 
Use **signedRoute()**
link a: https://a.com/document/829975?signature=2cea3458eace30995eb5456474b5cf3465e4889c4d514def86c1ea9d1d40fb31

**B domain**
link b: https://b.com/document/829975?signature=2cea3458eace30995eb5456474b5cf3465e4889c4d514def86c1ea9d1d40fb31

now can not access link b.

I want to access link b,so what should I do?The premise is that the domain name B must be used.

**Comment by henzeb** [comment]
Redirect from domain A to a signed equivalent?

**Comment by rojtjo** [comment]
This happens because the signature includes the domain name. You could create the url by setting the `absolute` parameter to `false`:

```php
// Update the middleware on your route to use validate relative signed URLs
Route::middleware('signed:relative')
    ->get('my-route/{param}')
    ->name('my-route');

// Create a relative signed URL
$url = URL:: signedRoute('my-route', 'my-param', absolute: false);

// Prefix with the app URL (optional) 
$url = URL::to($url);
```
